components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -121,6 +121,8 @@ public class Utils {
 
     private static final String PROPERTY_PASSWORD_ERROR_MSG = "PasswordJavaRegExViolationErrorMsg";
 
+    private static final String AT_SYMBOL = "@";
```

Comment:
shall we rename this constant name? Eg: email Username identifier or something like that?

Reply:
Fixed with ab3decd79412eb74746363d8323bec7dbe0249a0